crates/bevy_asset/src/server/mod.rs

```diff
@@ -281,7 +281,7 @@ impl AssetServer {
     }
 
     #[must_use = "not using the returned strong handle may result in the unexpected release of the asset"]
-    pub(crate) async fn load_untyped_async<'a>(
+    pub async fn load_untyped_async<'a>(
```

Comment:
Now that it's public, we could add some docs and maybe point to the alternative API that "hides" the async.